src/test/java/io/knotx/commons/http/request/MultiMapCollectorTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.commons.http.request;
+
+import io.vertx.reactivex.core.MultiMap;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.knotx.commons.http.request.AllowedHeadersFilterTest.getPatternFilter;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class MultiMapCollectorTest {
+
+    @Test
+    @DisplayName("Expect collector correct result")
+    void expectCollectorCorrectResult() {
+        MultiMap multi = MultiMap.caseInsensitiveMultiMap();
+
+        List<String> contentType = Arrays.asList(DataObjectsUtilTest.CONTENT_TYPE_HTML);
```

Comment:
Let's have separate constant here for that - so that changing one test does not have impact on another.

Reply:
Done